pkg/cloudprovider/azure.go

```diff
@@ -256,13 +267,13 @@ OuterLoop:
 		}
 	}
 	ipConfigurations = append(ipConfigurations, newIPConfiguration)
-	networkInterface.IPConfigurations = &ipConfigurations
+	networkInterface.Properties.IPConfigurations = ipConfigurations
 	// Send the request
-	result, err := a.createOrUpdate(networkInterface)
+	err = a.createOrUpdate(networkInterface)
```

Comment:
```suggestion
	return a.createOrUpdate(networkInterface)
```

Reply:
Didn't add this since [this comment](https://github.com/openshift/cloud-network-config-controller/pull/166/commits/8d7fa811c2b01fc6e6cb794d1593745c3fdd4536#r2028300777) changes what is requested in this comment.